CONTRIBUTING.md

```diff
@@ -0,0 +1,97 @@
+Contributing to Rust-Lightning
+==============================
+
+The Rust-Lightning project operates an open contributor model where anyone is
+welcome to contribute towards development in the form of peer review, documentation,
+testing and patches.
+
+Anyone is invited to contribute without regard to technical experience, "expertise", OSS
+experience age or other social discriminant. Though developing cryptocurrencies demand a
+high-level bar of rigor, adversial thinking, thorough testing and risk-minimization.
+Any bug may cost real-money and so impact severely people lifes. That's said we're deeply
```

Comment:
s/Any bug may cost real-money and so impact severely people lifes./Any bug may cost users real money./
s/That's said/That said/